app/containers/messages.js

```diff
@@ -50,123 +56,129 @@ function MessagesContainer (props) {
                     {repeatedAuthor ? null : <Avatar name={message.key} />}
                   </div>
+                  <div className='messages__item__metadata'>
+                    {!repeatedAuthor &&
+                      <div onClick={onClickProfile.bind(this, user)} className='messages__item__metadata__name'>
+                        {user.name}
+                        {user.isAdmin() && <span className='sigil admin' title='Admin'>@</span>}
+                        {!user.isAdmin() && user.isModerator() && <span className='sigil moderator' title='Moderator'>%</span>}
+                        {renderDate(formattedTime)}
+                      </div>}
+                    <div className={repeatedAuthor ? 'text indent' : 'text'}>
+                      {enriched.content}
+                    </div>
+                  </div>
                 </div>
-                <div className='messages__item__metadata'>
-                  {repeatedAuthor ? null : <div onClick={onClickProfile.bind(this, user)} className='messages__item__metadata__name'>{user.name}{renderDate(formattedTime)}</div>}
-                  <div className={repeatedAuthor ? 'text indent' : 'text'}>{enriched.content}</div>
+              )
+            }
+            if (message.type === 'chat/emote') {
+              item = (
+                <div className='messages__item messages__item--emote'>
+                  <div className='messages__item__avatar'>
+                    <div className='messages__item__avatar__img' onClick={onClickProfile.bind(this, user)}>
+                      {repeatedAuthor ? null : <Avatar name={message.key} />}
+                    </div>
+                  </div>
+                  <div className='messages__item__metadata'>
+                    {repeatedAuthor ? null : <div onClick={onClickProfile.bind(this, user)} className='messages__item__metadata__name'>{user.name}{renderDate(formattedTime)}</div>}
+                    <div className={repeatedAuthor ? 'text indent' : 'text'}>{enriched.content}</div>
+                  </div>
                 </div>
+              )
+            }
+            return (
+              <div key={message.time + message.key}>
+                {showDivider && (
+                  <div className='messages__date__divider'>
+                    <h2> {formattedTime.long} <span>({printDate.fromNow()})</span> </h2>
+                  </div>
+                )}
+                {item}
               </div>
             )
-          }
-          return (
-            <div key={message.time + message.key}>
-              {showDivider && (
-                <div className='messages__date__divider'>
-                  <h2> {formattedTime.long} <span>({printDate.fromNow()})</span> </h2>
-                </div>
-              )}
-              {item}
-            </div>
-          )
-        })}
-      </div>
+          })}
+        </div>
+        <div
```

Comment:
added this div so it can be ref to scrolled down to

Reply:
@khubo maybe this comment can be inlined with the source code, as well? i bet someone in the future will be like "what is this div here for?!"